extensions/replication/ReplicationQueuePopulator.js

```diff
@@ -52,7 +52,18 @@ class ReplicationQueuePopulator extends QueuePopulatorExtension {
                      `${queueEntry.getBucket()}/${queueEntry.getObjectKey()}`,
                      JSON.stringify(entry));
 
-        this._incrementMetrics(entry.bucket, queueEntry.getBytesMetric());
+        const repSites = queueEntry.getReplicationInfo().backends;
+        const sites = repSites.reduce((store, entry) => {
```

Comment:
A cleaner way for this is to use `Array.filter` method:

```
const sites = repSites.filter(entry => entry.status === 'PENDING')
```

Or you may also simply add this check in the below `forEach` loop directly.